### PR TITLE
feat: Ecosystem config preview

### PR DESCRIPTION
**Pull request by Jabher**
## Description
This PR introduces basic renderer for Ecosystem to improve in future.
Also, this adds lido-contents as submodule (with `lido-landing` symlink) for ease of debug
## Demo
Screenshot 2023-07-19 at 00 16 55

### CODE REVIEW

File: .gitmodules
Comment (Slavik191) [inline .gitmodules:1]: What exactly is the submodule here for?

File: features/cms/initialize/collections/lidoLanding/index.ts [hunk 1]
```diff
@@ -6,9 +6,15 @@ import { faq } from "./files/faq";
...
+export const lidoLandingCollection: CmsCollection = {
   name: "lido-landing",
   label: "Lido landing",
   files: [scorecard, ecosystem, faq, validators, termsOfUse, audits],
 };
+
+
+export const registerEcosystemRenderer = () => {
+  CMS.registerPreviewTemplate(ecosystem.name, EcosystemConfigPreview);
+}
```
Comment (Slavik191) [inline features/cms/initialize/collections/lidoLanding/index.ts:20]: it is also better move in 'features/cms/preview/ecosystemConfig' (
into one file)

File: features/cms/initialize/initialize.ts [hunk 2]
```diff
@@ -23,11 +24,12 @@ export const initializeCMS = () => {
...
-        ecosystemConfig,
-        validatorsProjects,
+        lidoLandingCollection,
+        ecosystemProjectsCollection,
+        ecosystemConfigCollection,
+        validatorsProjectsCollection,
       ],
     },
   });
+  registerEcosystemRenderer();
```
Comment (Slavik191) [inline features/cms/initialize/initialize.ts:34]: Move the function call to pages/admin.tsx

File: features/cms/initialize/collections/lidoLanding/EcosystemConfigPreview.tsx
Comment (Slavik191) [inline features/cms/initialize/collections/lidoLanding/EcosystemConfigPreview.tsx:1]: does not display information about "MOST POPULAR"

File: yarn.lock [hunk 3]
```diff
@@ -7710,10 +7711,10 @@ typed-array-length@^1.0.4:
     for-each "^0.3.3"
     is-typed-array "^1.1.9"
 
-typescript@4.9.4:
-  version "4.9.4"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.4.tgz#a2a3d2756c079abda241d75f149df9d561091e78"
-  integrity sha512-Uz+dTXYzxXXbsFpM86Wh3dKCxrQqUcVMxwU54orwlJjOpO3ao8L7j5lH+dWfTwgCwIuM9GQ2kvVotzYJMXTBZg==
+typescript@5.1:
+  version "5.1.6"
+  resolved "https://registry.npmjs.org/typescript/-/typescript-5.1.6.tgz#02f8ac202b6dad2c0dd5e0913745b47a37998274"
```
Comment (kozlovzxc-at-lido) [inline yarn.lock:7716]: Why did you switch to npmjs? 🤔

**Review by kozlovzxc-at-lido — APPROVED** [review]

**Pull request closed without merging**